packages/cf-component-form/src/FormTheme.js

```diff
@@ -0,0 +1,10 @@
+import { checkBaseTheme } from 'cf-style-const';
+
+export default baseTheme => {
+  checkBaseTheme(baseTheme, 'FormTheme');
+  return {
+    background: 'white',
+    border: `1px solid ${baseTheme.colorGrayLight}`,
```

Comment:
We should call this atomization rather than DCE. This would be even better:

```js
...expandBorder('1px solid black')
```

output

```js
borderTopColor: 'black',
borderLeftColor: 'black',
borderBottomColor: 'black',
borderRightColor: 'black',
borderTopColor: 'black',
borderLeftStyle: 'solid',
borderBottomStyle: 'solid',
borderRightStyle: 'solid',
borderTopWidth: '1px',
borderLeftWidth: '1px',
borderBottomWidth: '1px',
borderRightWidth: '1px'
```